### DISCUSSION TITLE
Would the deadline for final submission be extended or would it remain the same?

### DISCUSSION THREAD

**Discussion by SadmanIslamAlif** [larics/icuas22_competition#46] [Q&A]
Greetings dear organizers 
1. Would the deadline be extended for final submission? As we have our finals coming soon to take preparations so we needed to know it. 
2. And would the evaluation period start from 15th of April? 
3. Will there be multiple evaluations? 
4. after the deadline will the evaluation go through the final submissions? 
5. Should we book a window for the evaluation ?
6. After evaluation could we submit another solution?

**Comment by fpetric** [comment]
Hi @SadmanIslamAlif 

thanks for getting in touch! Let me get straight into it :)



> 1. Would the deadline be extended for final submission? As we have our finals coming soon to take preparations so we needed to know it.

As of now, we do not plan on extending the deadline for submission, since we need to leave enough time for the finalists to arrange their trips to Dubrovnik. Even if we do extend it (we will discuss it since a few teams asked), it would not be much longer than a week.



> 2\. And would the evaluation period start from 15th of April?

Evaluation period started already :) you can submit your solution at any time according to [these](https://github.com/larics/icuas22_competition/discussions/39) and [these](https://github.com/larics/icuas22_competition/discussions/43) instructions. Make sure to upload your team info too. 



> 3\. Will there be multiple evaluations?

Yes, we plan on running the evaluations frequently (we set ourselves the aim to do it daily for new submissions). So if you submit your solution on Monday, we will probably run the evaluation for it on Tuesday. That's the plan. 

> 4\. after the deadline will the evaluation go through the final submissions?

Not sure if I understand the question, but the idea for the final ranking is to be based on several of best runs from the evaluation period, whenever they happened. 



> 5\. Should we book a window for the evaluation ?

No, you just upload your solution to the folder you were shared via email, according to instructions [here](https://github.com/larics/icuas22_competition/discussions/39) and [here](https://github.com/larics/icuas22_competition/discussions/43). Our system detects changes in the shared folder, downloads your solution and tries to build it and run. 



> 6\. After evaluation could we submit another solution?

Yes, if you make improvements you just upload a new version of your solution and our system will detect the change and run the evaluation the next day. 

Hope this answers the question, if you need any more info let me know. :)
